rust-toolchain

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "nightly-2021-02-21"
+channel = "nightly-2021-08-01"
```

Comment:
Is this intentional? I don't see a corresponding bump in https://github.com/PureStake/moonbeam/pull/740/ -- do we want to keep them in sync?

Reply:
IT's already in master https://github.com/PureStake/moonbeam/blob/master/rust-toolchain